WalkingChallenge/Assets.xcassets/Aga Khan Foundation LOGO_raster_65.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "Aga Khan Foundation LOGO_raster_65.png",
```

Comment:
Similar with the `_raster_` infix.  Whats the `65` suffix for?

Reply:
I felt that the image was too big and i resized by 65%, turns out it didn't matter. The name was just an indicator to myself of the changes made.